cog.c

```diff
@@ -344,11 +344,9 @@ on_shutdown (CogLauncher *launcher G_GNUC_UNUSED, void *user_data G_GNUC_UNUSED)
 {
     g_debug ("%s: Platform = %p", __func__, s_options.platform);
 
-    if (s_options.platform) {
-        cog_platform_teardown (s_options.platform);
-        g_clear_object(&s_options.platform);
-        g_debug ("%s: Platform teardown completed.", __func__);
-    }
+    g_clear_object(&s_options.platform);
+
+    g_debug("%s: Platform destroyed.", __func__);
```

Comment:
Does it still make sense to have this debug log here? The platform can be destroyed elsewhere, maybe this should be "on_shutdown called" instead, or this statement moved to the finalize function?

Reply:
Let's remove this debugging statement, I think it's not particularly useful.